tests/unit/test_data.py

```diff
@@ -178,13 +178,17 @@ def test_download_ok(self, data_mock):
 
         Data.download(data_mock)
         data_mock.resolwe._download_files.assert_called_once_with(
-            ["123/file1.txt", "123/file2.fq.gz"], None
+            files=["123/file1.txt", "123/file2.fq.gz"],
+            download_dir=None,
+            custom_file_names=None,
         )
 
         data_mock.reset_mock()
         Data.download(data_mock, download_dir="/some/path/")
         data_mock.resolwe._download_files.assert_called_once_with(
-            ["123/file1.txt", "123/file2.fq.gz"], "/some/path/"
+            files=["123/file1.txt", "123/file2.fq.gz"],
+            download_dir="/some/path/",
+            custom_file_names=None,
         )
 
```

Comment:
Would be nice if you add a test as well. If you need help, don't hesitate to ask.